.env.example

```
@@ -1,6 +1,15 @@
 # Axon Rust Environment Template
 # Copy to .env and set values for your environment.
 
+# --- Host paths (required for session ingestion) ---
+# [REQUIRED for sessions ingest] Absolute path to the host user's home directory.
+# Used to bind-mount AI session history into the axon-workers container.
+# Example: HOST_HOME=/home/yourname
+HOST_HOME=/home/yourname
+# [OPTIONAL] Root directory for all axon persistent data on the host.
+# Defaults to ./data if unset.
+AXON_DATA_DIR=./data
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AXON_DATA_DIR key should go before the HOST_HOME key

(UnorderedKey)